pori_python/ipr/inputs.py

```diff
@@ -505,14 +505,14 @@ def preprocess_tmb(
 
     # genomeTmb
     # SDEV-4811 - mutation burden is now expected to be uploaded in genomeTmb as mutations/megabase
-    if genomeTmb != None and genomeTmb != "":
+    if genomeTmb:
```

Comment:
Can't genomeTmb ever be 0.0 ?

Reply:
It's a string.
And the default tmb_val is 0.0 anyway